yarn.lock

```diff
@@ -1641,10 +1641,10 @@ chrome-trace-event@^1.0.2:
   dependencies:
     tslib "^1.9.0"
 
-chromedriver@^77.0.0:
-  version "77.0.0"
-  resolved "https://registry.yarnpkg.com/chromedriver/-/chromedriver-77.0.0.tgz#bd916cc87a0ccb7a6e4fb4b43cb2368bc54db6a0"
-  integrity sha512-mZa1IVx4HD8rDaItWbnS470mmypgiWsDiu98r0NkiT4uLm3qrANl4vOU6no6vtWtLQiW5kt1POcIbjeNpsLbXA==
+chromedriver@^78.0.0:
+  version "78.0.1"
+  resolved "https://artifactory.in.here.com/artifactory/api/npm/here-node/chromedriver/-/chromedriver-78.0.1.tgz#2db3425a2cba6fcaf1a41d9538b16c3d06fa74a8"
```

Comment:
The public yarn.lock must not point to our internal artifactory

Reply:
Ooops. Any idea how this happend? I just run `yarn install` to update the `yarn.lock`. And there is no `.npmrc` in this project?